### PR TITLE
Update controller service name

### PR DESCRIPTION
**Pull request by pthangad**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/main/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/main/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/main/CONTRIBUTING.md)
for more details._

# Release Notes

```
release-note
```

### CODE REVIEW

File: openshift/e2e-add-service-account.py [hunk 1]
```diff
@@ -9,6 +9,9 @@
 import sys
 data = list(yaml.safe_load_all(sys.stdin))
 for x in data:
-    if x['kind'] in ('PipelineRun', 'TaskRun'):
-        x['spec']['serviceAccountName'] = sys.argv[1]
+    try:
+        if x['kind'] in ('PipelineRun', 'TaskRun'):
+            x['spec']['serviceAccountName'] = sys.argv[1]
+    except TypeError:
```
Comment (chmouel) [inline openshift/e2e-add-service-account.py:15]: it's a bit of broad exception catch, what is the error? no spec in x or no kind in x or no sys.argv[1] ?

Comment (chmouel) [reply openshift/e2e-add-service-account.py:15]: a better fix

```diff
diff --git a/openshift/e2e-add-service-account.py b/openshift/e2e-add-service-account.py
index 6e4c822..c8887c0 100755
--- a/openshift/e2e-add-service-account.py
+++ b/openshift/e2e-add-service-account.py
@@ -9,6 +9,8 @@ import yaml
 import sys
 data = list(yaml.safe_load_all(sys.stdin))
 for x in data:
+    if not x:
+        continue
     if x['kind'] in ('PipelineRun', 'TaskRun'):
         x['spec']['serviceAccountName'] = sys.argv[1]
 print(yaml.dump_all(data, default_flow_style=False))
```

Comment (praveen4g0) [reply openshift/e2e-add-service-account.py:15]: Agree!

**Pull request closed without merging**